webdev2.html

```diff
@@ -0,0 +1,112 @@
+
+    <script>
+        // initialize the map
+        var map = L.map('map').setView([46.7867, -92.1005], 11);
+
+        // load a tile layer
+        // loading the base layer of map tiles using a URL template
+        // this template ({z}/{x}/{y}) allows leaflet to locate tiles with the
+        // correct zoom, x, and y coordinates
+        L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', { attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>' }).addTo(map);
+
+        var airportIcon = L.icon({
+            iconUrl: '../images/airport.png',
+            iconSize: [50,40]
+          });
+          var customLayer = L.geoJson(null  ,{
+              pointToLayer: function(feature,latlng){
+                return L.marker(latlng,{icon: airportIcon});
+              }
+          });
+          var airportsLayer = omnivore.csv('airports.csv', null, customLayer).addTo(map);
```

Comment:
Great work getting this to function.
But put more comments and spacing into your code to make it more comprehensible.

It looks like the reason the airports are not showing up is that you are missing the `airport.png` image file.